R/oauth-refresh.R

```diff
@@ -17,6 +17,9 @@ refresh_oauth2.0 <- function(endpoint, app, credentials) {
   )
 
   response <- POST(refresh_url, body = body, encode = "form")
+  if (has_oauth2.0_error(response)) {
+    return(NULL)
```

Comment:
But maybe you're trying to separate this into known errors and unknown errors? I'd like to hear your thinking about this.